inst/rmd/h/anchor-sections/anchor-sections.css

```diff
@@ -1,5 +1,4 @@
 /* Styles for section anchors */
 a.anchor-section {margin-left: 10px; visibility: hidden; color: inherit;}
-a.anchor-section::before {content: '#';}
```

Comment:
What about keep this line so that people can easily find the default behavior?
We can override the rule by including other CSS files.

Reply:
an remove `anchor-section-hash.css` ? 
I thought regrouping everything was clearer (I added a font-size rule too)

Regarding "finding behavior", I expect it to be easier with the documentation I made (in NEWS and in `?html_document`) that looking through the files. 

No strong opinion on this, even if I prefer separating the styling in files.

Reply:
Yes, your document is clear.
I just prefer self-explaining code.
Mine is also not a strong opinion.
You can ignore.

Reply:
By self explaining you mean having the default `anchor-sections.css` contains the whole default style for hash character ? instead of having a default which is a combination of two files ?

Reply:
Yes.

Reply:
https://github.com/rstudio/rmarkdown/pull/1964#discussion_r533472286